tools/hub.py

```diff
@@ -0,0 +1,15 @@
+from logging import info, basicConfig, INFO
```

Comment:
Why not moving this file into yolox.utils dir?

Reply:
Done